filecoin-proofs/src/api/mod.rs

```diff
@@ -117,20 +117,20 @@ pub fn generate_piece_commitment<T: std::io::Read>(
         ));
     }
 
-    let n = UnpaddedBytesAmount(n as u64);
+    let n_u = UnpaddedBytesAmount(n as u64);
```

Comment:
nit: this is really error-prone for the future `n` and `n_u`. I would name it e.g. `unpadded`.